### ISSUE TITLE
REFERENCE: Daemons in Heretic Astartes catalogues

### ISSUE THREAD

**Issue by sanguissaevus** [BSData/wh40k#2445]
The below also applies to Daemon entries in other Heretic Astartes catalogues.

Death Guard codex for warhammer 40000 is missing most of its daemon unit selections such as nurglings, plaguebearers, plague drones, beasts of nurgle.

**Comment by Mad-Spy** [comment]
There is no such thing as a "Death Guard" Daemon. Adding Daemons to a DG detachment will cause that detachment to no longer be DG as not all models have the DEATH GUARD faction keyword.

To help users to make better lists, Daemons have been removed from all HERETIC ASTARTES catalogues.

If you want to add daemons to your army, add a separate (or child) detachment of Daemons.

This will have the added benefits of giving you access to all Nurgle Daemons rather than just the limited selection from the DG Codex, and access to Daemon-specific  Traits, etc.

**Comment by sanguissaevus** [comment]
Incorrect.  The death guard codex has specific daemons in it and others (that you cant have access to without the daemon codex) not included.  These "death guard daemons" do not have access to daemon codex specific rules such as the strategems and locus'.  Ommiting them from the death guard may make your job easier but it is still  incorrect.  This resolution you have created makes the lists invalid without clear explanation to other players.  Statements like, "No, those nurglings are actually appart of my death guard detachment and not the daemon detachment the list is showing and so makes my death guard detachment valid.  Just pretend i dont have the daemon detachment my Battlescribe list says i have and it makes sense."  That example, and thereby your reasoning, defeats much of the whole reason for battlescribe, which is to have an organized legitimate list that communicates the game rules clearly.  

In conclusion, I sympathize with your desire to make your job easier but it is, in the end, cutting corners and thereby creating a sloppy final product.  Please correct this error.  Thanks.

**Comment by skonk** [comment]
None of the Daemon's units listed in the Death Guard codex have the Heretic Astartes or Death Guard faction keywords. As such, if you add them to the same Detachments as your Death Guard units, the Detachment stops being a Death Guard Detachment and it would invalidate the use of the special rules associated with the Death Guard army. The same goes for the Daemons units inside the CSM and TS codexes.

These Daemon units are included in the book as a convenience for use when summoning; which is why they don't need other faction keywords and/or access to the Codex Daemons special rules since they will never be inside a Detachment, due to being summoned to the table.

But, when using Daemons, either for Summoning or for adding to Detachments inside your army; there is nothing preventing a Death Guard (or Chaos Space Marines in general) player from simply using those units directly out of Codex: Chaos Daemons.

The Daemons units inside the Chaos Space Marines (CSM, DG, TS) books can literally be ignored for all game purposes, if you own the Daemons codex. You can just use them from the Daemons book.

As a result; including them in the DG/CSM/TS data files just creates duplicates and leads BattleScribe users who are not fully up on how Keywords work in relation to Detachment rules to wrongly believe that they can use these Daemons units inside their main Detachments while still using the full compliment of their Army rules; causing a lot of people to unwittingly cheat.

But as has already been pointed out; if you want to add detachments to use those daemons, or if you still insists on putting them inside the same detachment as your Death Guard units (and thus lose your Death Guard special rules) then you can just add a Daemons detachment INSIDE the Death Guard detachment, add the chosen units and the detachment will Validate as if it was a single detachment.

**Comment by sanguissaevus** [comment]
Very well.  My mistake.  They dont have the death guard keyword.  Very confusing.  Never looked cause they were in the codex so I assumed they were death guard.  Carry on.

**Comment by skonk** [comment]
A lot of people assume the same thing to be honest; thats part of the rationale for removing them here. 

Games Workshop made a huge mistake by mixing these units into the main unit pages instead of having them at the back of the book and clearly labelled as "for summoning only"; and it's understandably confused the hell out of a lot of people.
